azure-stack-go-sdk/azure-stack-go-sdk-profiles.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Installing and use Azure Go SDK profiles on Azure Stack
+description: How to install and use Azure Go SDK profiles on Azure Stack environment.
+keywords: azure, azure stack, sdk, go, golang, azure sdk
+author: seyadava
+ms.author: seyadava
+ms.date: 03/22/2018
+ms.topic: article
+ms.devlang: go
+manager: knithinc
+---
+
+# What is a profile?
+A profile is a combination of different resource types with different versions from different services. Using a profile will help you mix and match between various resource types. Profiles can provide:
+
+* Stability for your application by locking to specific API versions; and/or
+* Compatibility for your application with Azure Stack and regional Azure datacenters.
+
+In Azure Go SDK, profiles are available under the profiles/ path, with their version in the “YYYY-MM-DD” format. Right now, the latest Azure Stack profile version is “2017-03-09”. To import a given service from a profile, you need to import its corresponding module from the profile. For example, to import “Compute” service from “2017-03-09” profile:
+
+```go
+import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/compute/mgmt/compute"
+```
+
+# Install Azure Go SDK on Azure Stack
+1. Follow the official instructions to install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+2. Follow the official instructions to install [Go](https://golang.org/dl/). Note that to use profiles you need to install Go version 1.9 or newer.
+3. Install Go SDK and its dependencies by running the following bash command:
+
+    ```bash
+    go get -u -d github.com/Azure/azure-sdk-for-go/...
+    ```
+    * Full details for Azure Go SDK Installation are [here](https://docs.microsoft.com/en-us/go/azure/azure-sdk-go-install#get-the-azure-sdk-for-go)
+    * Azure Go SDK is publicly available at [Go SDK GitHub](https://github.com/Azure/azure-sdk-for-go)
+    * Azure Go SDK uses Go-AutoRest modules to send requests to Azure resource provider endpoints. To use Azure services in code, in addition to Azure Go SDK, modules must be imported from [Go-AutoRest](https://github.com/Azure/go-autorest). More details about the most common Go-Autorest packages can be found [here](https://docs.microsoft.com/en-us/go/azure/azure-sdk-go-install#including-the-azure-sdk-for-go-in-your-project). For a complete list of the available modules in Go-AutoRest go [here](https://godoc.org/github.com/Azure/go-autorest).
+
+# How to use GO SDK profiles on Azure Stack
+To run a sample of Go code on Azure Stack:
+
+1. Install Azure Go SDK and its dependencies (see previous section)
+2. Get metadata information from resource manager endpoint. Resource manager metadata endpoint returns a JSON file with required information to run Go samples. To retreive the metadata, open a web browser in your Azure Stack environment and use the url: `ResourceManagerURL`/metadata/endpoints?api-version=1.0
```

Comment:
This section contains multiple sentence fragements; `Resource manager metadata endpoint...`, `Sample JSON file...`. These sentences are also missing articles.

Reply:
The entire URL for the stack environment should be in ` `` ` quotes. `ResourceManagerURL` should be written in a style that indicates it is a user-provided value.